### DISCUSSION TITLE
Choose Native Filter in URL

### DISCUSSION THREAD

**Discussion by mblanloeil** [apache/superset#18830] [General]
Hi 

It is possible to preselect a native filter by URL ?
Sample : 

We have à dashboard accessible with an URL like : www.mysuperset.com/dashnoard1 
In this dashboard we have a lot of native filter and we want create custom URL for choose a native filter : www.mysuperset.com/dashnoard1?filter1=test&filter2=myFilter2 etc .. 

Thanks!

**Comment by falgunifb** [comment]
@mblanloeil are you able to resolve this

**Comment by Thblink1995** [comment]
I would like to know the answer to that as well
